### PR TITLE
New: added commit log for 0.9.9

### PR DESCRIPTION
**Pull request by Dogild**

### CODE REVIEW

File: source/cappuccino-release-notes.markdown [hunk 1]
```diff
@@ -3,6 +3,262 @@ title: Cappuccino Release Notes
...
+* New: `CPAppearance` and `CPVisualEffectView` support.
+* Fixed: optimization of the `CPTableView`.
+
+
+Foundation and Core
+-------------------
+
+* (630aa53) New: `CPCache` has been added to `Foundation`.
+* (58afc92) New: timeout and cache feature in `CPURLRequest`.
+* (806be9) New: `CPURLConnectionDelegate` protocol.
```
Comment (primalmotion) [inline source/cappuccino-release-notes.markdown]: I guess it's `806be95`

**Comment by primalmotion** [comment]
can you start the bullets with a dash? I guess the stars are screwing up the md

**Comment by Dogild** [comment]
We have used the star for the previous release as well

**Comment by primalmotion** [comment]
or maybe it's just github lost in that monstrous change log :) works fine on an external editor

**Pull request merged**